src/main/resources/jupyter/cluster-site.conf

```diff
@@ -24,6 +24,19 @@
     ProxyPass / http://127.0.0.1:8001/ retry=10
     ProxyPassReverse / http://127.0.0.1:8001/
 
+    # Welder
+
+    RewriteCond %{HTTP:Upgrade} =websocket
```

Comment:
I'm guessing this block can be removed.  Welder doesn't need to handle websockets.

Reply:
yeah it can be removed